MVVM Module/Tests/Feed UI/FeedUIIntegrationTests.swift

```
@@ -280,7 +280,20 @@ final class FeedUIIntegrationTests: XCTestCase {
 		}
 		wait(for: [exp], timeout: 1.0)
 	}
-	
+    
+    func test_loadFeedCompletion_rendersErrorMessageOnErrorUntilNextReload() {
```

Same here - It's missing a test: should hide the error when the user taps on the error view.